docs/reference/dapis/chains/chains-list.md

```diff
@@ -36,7 +42,9 @@ They will not render as HTML but rather text or you get an error. No idea why!
 </div>
 
 <script setup lang="ts">
-    import chains from './chains.json';
+  import {ref} from 'vue';
```

Comment:
Maybe this import could use a comment?

Reply:
I added `ref` with the intent to use a `ref` but changed my mind. Of course I would forget to remove the import.